libs/shared/utils/src/lib/sleep.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const sleep = (ms = 500) =>
```

Comment:
please use delay 😉

Reply:
https://github.com/mudita/mudita-center/blob/develop/libs/shared/utils/src/lib/delay.ts

Reply:
🤦 
Fixed in [39ad362](https://github.com/mudita/mudita-center/pull/2204/commits/39ad36253f9dbe8778590a52869894aba3bc5a5d)